pyrk/th_system.py

```diff
@@ -83,8 +133,9 @@ def convection(self, t_b, t_env, h, A):
         denom = (1.0/(h*A))
         return num/denom
 
-    def conduction(self, t_b, t_env, L, k, A):
+    def conduction(self, t_b, t_env, r_b, r_env, k):
```

Comment:
If you are going to change the input parameters, you must also change the docstrings so that you communicate their new purpose.